test/cmdlineTests.sh

```diff
@@ -66,11 +72,22 @@ function ask_expectation_update()
 {
     if [ $INTERACTIVE ]
     then
+        if [ $AUTOUPDATE = "true" ]
+        then
+            echo "$newExpectation" > $expectationFile ;
+            return
+        fi
```

Comment:
```suggestion
        if [[ $AUTOUPDATE == true ]]
        then
            echo "$newExpectation" > "$expectationFile"
            return
        fi
```

even better:
```suggestion
        [[ $AUTOUPDATE == true ]] && echo "$newExpectation" > "$expectationFile"
        [[ $AUTOUPDATE == true ]] && return
```

And same in one more case below.